azurelinuxagent/common/cgroup.py

```diff
@@ -30,6 +30,55 @@ class CGroup(object):
     def __init__(self):
         pass
 
+    def _get_cgroup_file(self, hierarchy, file_name):
```

Comment:
Are `hierarchy` and `controller` equivalent? If so, can we decide on one name?

Reply:
yes, i sync'ed with varad on this... at this point I just moved the existing code here... varad will take it from here and decide whether we need the code, refactor as needed, etc.